### DISCUSSION TITLE
Organization Seats Conflict

### DISCUSSION THREAD

**Discussion by edward-data-sci** [community/community#23280] [New to GitHub]
<p>My organization pays for 18 seats, and is currently using 13. This is indicated when I manage the members and collaborators in our organization. GitHub settings also says that I have 5 seats available. However, when I send invitations to new collaborators, the invitations fail, stating that we have no more seats. Where do I go, or what do I do to resolve these two conflicting sets of information on the organization account?</p>

**Comment by wordsoup** [comment, marked as answer]
<p>Hi <a class="mention" href="/edward-data-sci">@edward-data-sci</a>,</p>
<p>If your organization’s seat count is not automatically adjusting as expected based on our <a href="https://help.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/about-per-user-pricing#how-many-licenses-do-i-need-for-my-organization" title="How many licenses do I need?" rel="nofollow noopener">guidelines</a>, we will need you to work with our private <a href="https://support.github.com/contact" title="Contact Support" rel="nofollow noopener">Support</a> team to take a look at your specific account. If you’ve already submitted a ticket to our Support team, thanks for your patience! They’ll get to you as soon as they can.</p>
